code/game/machinery/doors/showcase.dm

```diff
@@ -0,0 +1,176 @@
+/obj/machinery/door/blast/shutters/glass
+	name = "showcase"
+	icon = 'icons/obj/doors/showcase.dmi'
+	icon_state = "closed"
+	health = 200
+	maxhealth = 200
+	layer = 4.2
+	var/broken = 0
+	var/empty = 0
```

Comment:
Please replace this variable values with TRUE/FALSE.
And also I think that we should rename it to have_glass or something like that to clarify behaviour.